testcases/kernel/syscalls/fallocate/fallocate01.c

```diff
@@ -84,6 +84,17 @@
  *		Cleanup the temporary folder
  *
 *************************************************************************/
+/*
+ * Patch Description:
+    Test Failure reason in SGX-LKL:
+    Test is happening on temporary file created under /tmp, on which fallocate system call is failing.
+    Please refer git issue : https://github.com/lsds/sgx-lkl/issues/734
+
+ * Workaround to fix the issue:
+    fallocate system call is failing on temporary file created under /tmp directory.
+    So modified the tests to create directory in root filesystem.
+   TODO: Enable tst_tmpdir once git issue 734 is fixed
```

Comment:
"git issue 734" is unclear. Please update to be the full url to the issue in question.

Reply:
Or perhaps "aforementioned"

Reply:
@SeanTAllen Done